rosidl_generator_py/CMakeLists.txt

```diff
@@ -49,6 +49,7 @@ if(BUILD_TESTING)
   include(cmake/register_py.cmake)
   include(cmake/rosidl_generator_py_get_typesupports.cmake)
 
+  set(rosidl_generator_py_FOUND "1")
```

Comment:
It's probably worth re-iterating the reason for this in a comment too

Reply:
Added comment in 8334b82